docs/Web-Flow-1.10.0.md

```diff
@@ -1,8 +1,40 @@
-# Migration from 1.9.0 to 1.10.0
+# Migration from 1.9.0 to 1.10.x
 
-This guide contains instructions for migration from PowerAuth WebFlow version `1.9.x` to version `1.10.0`.
+This guide contains instructions for migration from PowerAuth WebFlow version `1.9.x` to version `1.10.x`.
 
 
+# 1.10.0
+
 ## Database Changes
 
 Added table `shedlock` to prevent execution of the same task from another node.
+
+# 1.10.1
```

Comment:
🤔 We have not yet split the migration guides by bugfix version, but why not? Just highlighting that the filename is `docs/Web-Flow-1.10.0.md`.

Reply:
Yes, but as we are going to do several patchsets, we should start using the migration in an incremental way..